fontforge/scstyles.c

```diff
@@ -5185,7 +5183,8 @@ static void SerifRemove(SplinePoint *start,SplinePoint *end,SplineSet *ss) {
 	SplineFree(spnext->prev);
     }
     start->next = end->prev = NULL;
-    start->nonextcp = end->noprevcp = true;
+    start->nextcp = start->me;
```

Comment:
If these are needed its because something doesn't re-splice the removed section of the contour, which is unlikely, but I put them back with a comment anyway.